packages/frontend-v2/components/Sidebar/SectionRequirement.tsx

```diff
@@ -127,6 +127,7 @@ const SectionRequirement: React.FC<SidebarRequirementProps> = ({
   const renderSection = (requirement: Section) => {
     return (
       <SidebarSection
+        isValid
```

Comment:
this prop was missing in main but required. @AlpacaFur @jia614, seems like we don't handle nested sections just yet, is this fine?

Reply:
Yeah, that should be fine for now, we can take another look when we want to handle nested sections. (I'm not fully sure why the lint didn't catch the missing prop though 👀).